submissions/cynthiaNwakaeme/HTML/index.html

```diff
@@ -13,18 +13,16 @@
         
```

Comment:
@Cynthia-Berry Is this your product page?

Reply:
@Cynthia-Berry You did not link your signup form?

Reply:
Hello @ToluAkin .
I have effected the changes as requesterd.
Kindly approve my PR.
Thank you so much